README.md

```diff
@@ -40,14 +42,20 @@ npx @readme/micro@v2.4.0 './*{yaml,yml,json}' --key=$README_MICRO_SECRET
 ### As a Bitbucket Pipeline
 
 ```yaml
-image: node:18
-
+definitions:
+  steps:
+    - step: &readme-micro
+        name: ReadMe Micro
+        image: node:18
+        script:
+          - npx @readme/micro@v2.4.0 '**/*.{yaml,yml,json}' --key=$README_MICRO_SECRET
+
+# Run Pipeline to sync OpenAPI files for every push to the `main` branch
 pipelines:
   branches:
-    # Run Pipeline to sync OpenAPI files for every push to the `main` branch
     main:
```

Comment:
since we know [Bitbucket still defaults to `master`](https://jira.atlassian.com/browse/BCLOUD-20212), should we still keep this as `main` (which should be the default – à la GitHub and git)?

Reply:
I dont really think it matters here since these are just examples, and integration is likely to happen from micro where this is pulled in from the API.

Reply:
Also bitbucket is inconsistent here... Input placeholder says main, when it's empty it defaults to master 🤷‍♂️